# overwrite obsolete -openmp from ParaFly configure script.
# Bart Oldeman, Calcul Quebec, Compute Canada
diff -ur trinityrnaseq-Trinity-v2.8.4.orig/trinity-plugins/Makefile trinityrnaseq-Trinity-v2.8.4/trinity-plugins/Makefile
--- trinityrnaseq-Trinity-v2.8.4.orig/trinity-plugins/Makefile	2018-09-12 10:08:03.000000000 +0000
+++ trinityrnaseq-Trinity-v2.8.4/trinity-plugins/Makefile	2018-12-07 19:57:49.677383177 +0000
@@ -21,7 +21,7 @@
 
 parafly_target:
 	tar -zxvf ${PARAFLY_CODE}.tar.gz && \
-	cd ${PARAFLY_CODE} && sh ./configure --prefix=`pwd` CXX=$(CXX) CC=$(CC) CFLAGS="-fopenmp" CXXFLAGS="-fopenmp" && $(MAKE) install && \
+	cd ${PARAFLY_CODE} && sh ./configure --prefix=`pwd` CXX=$(CXX) CC=$(CC) CFLAGS="-fopenmp" CXXFLAGS="-fopenmp" && $(MAKE) AM_CXXFLAGS="-fopenmp" install && \
 	cp bin/ParaFly ../BIN/
 
 
